packages/eslint-plugin-next/src/rules/no-before-interactive-script-outside-document.ts

```diff
@@ -30,19 +30,17 @@ export = defineRule({
         scriptImportName = node.local.name
       },
       JSXOpeningElement(node) {
+        const rootDir = path.resolve(__dirname, '../../../../../')
```

Comment:
This seems like it's resolving to an arbitrary path which won't be valid once installed outside of the monorepo. Could we start with adding a failing test case?

Reply:
Understood. Working on it.

Reply:
@ijjk Requesting for a review.